include/glow/Graph/Graph.h

```diff
@@ -173,26 +176,34 @@ class Module final {
   ///@{
 
   Placeholder *createPlaceholder(ElemKind T, llvm::ArrayRef<size_t> dims,
-                                 llvm::StringRef name, bool isTrainable);
+                                 llvm::StringRef name, bool isTrainable,
+                                 const std::string layout = ANY_LAYOUT);
```

Comment:
IIRC, the linter suggested making all such `const std::string layout` arguments `const std::string &layout`.

Reply:
solved, but the main linter issue i had is in switch-case statements where the internal linter disagreed with the external linter :)